src/proof/PGBuild.C

```diff
@@ -805,11 +805,11 @@ clauseid_t ProofGraph::dupliNode( RuleContext& ra )
 	clauseid_t v_id = ra.getV();
 	ProofNode* w = getNode( ra.getW() );
 	assert(w);
-	unsigned num_old_res = w->getNumResolvents();
+    unsigned num_old_res = w->getNumResolvents(); (void)num_old_res;
```

Comment:
Problem with indentation?

Reply:
It looks like it is problem of tabs vs spaces.
The old code was tabs, new uses spaces. My IDE represents the length of tab differently than github.
We should also have a style rule for this. Would 4 spaces per indentation level be a good rule? I prefer spaces to tabs.

Reply:
Fixed. Whole file now uses 4 spaces per indentation level.